typescript.md

```diff
@@ -191,6 +191,37 @@ from JS).
   }
   ```
 
+* Prefix `private` and `protected` fields with `_`:
+
+  > Explanation:
+  > * Underscore specifies visual difference from public fields. So, you don't need to read the
+  >   field's declaration to determine its visibility.
+  > * Minimizes name conflicts with other public fields: after adding public field with the same
```

Comment:
I think the word 'other' here introduces ambiguity.  'with other' makes me think that private fields are included with public fields in the same category. I think it will be more clear without it.

Reply:
You are right. Fixed.